windows/runner/utils.cpp

```diff
@@ -0,0 +1,64 @@
+#include "utils.h"
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase